admin/projectsConfig.mjs

```diff
@@ -0,0 +1,55 @@
+const GRAPHQL_FIELD_STUDY = 'Study';
+const GRAPHQL_FIELD_PARTICIPANT = 'Participant';
+
+// Reminder: by arranger standards, project id must be lowered case.
+const PROJECTS_IDS = {
+    migration_test_qa: 'migration_test_qa',
+};
+
+const commonMutations = [
```

Comment:
Ideally, i think the fields that need to be changed to  `isArray=true` should also be abstracted from the code in my opinion.